src/canvas/StaticCanvas.ts

```diff
@@ -323,14 +332,11 @@ export class StaticCanvas<
    * @param {Boolean}       [options.cssOnly=false]       Set the given dimensions only as css dimensions
    */
   setDimensions(
-    dimensions: Partial<TSize>,
-    { cssOnly = false, backstoreOnly = false }: TCanvasSizeOptions = {}
+    dimensions: Partial<TSize | CSSDimensions>,
+    options: TCanvasSizeOptions = {}
```

Comment:
question about this
What happens if I pass a css dimension string with option cssOnly = false?

I think we should change the second arg to be a string option or merge it into the first arg.
`setOnly?: 'css' | 'backstore'`

Reply:
Or better off 
```
css?: boolean = true,
backstore?: boolean = true,
```

OR add
```
cssWidth?: whatever,
cssHeight?: whatever,
```

Reply:
I prefer the last, explicit and clear

```
width: number,
height: number,
cssWidth?: whatever,
cssHeight?: whatever,
```

Reply:
yes with 4 width/height would be nice, but would be another breaking change.
Let's have this as a fix of the current situation and i ll open a different PR with the breaking change

Reply:
maybe i can just add cssWidth and cssHeight and deprecate the rest.

Reply:
> What happens if I pass a css dimension string with option cssOnly = false?

This has been solved now with stricter typings